sdk/test-utils/perf/src/utils/profiling.ts

```diff
@@ -0,0 +1,45 @@
+import { Session } from "node:inspector";
+import * as fs from "fs-extra";
+
+export async function runWithCpuProfile(
```

Comment:
Might not need to export this.

Reply:
removed the other wrapper for better readability, exporting this now.